Core/Utilities/DErrno.py

```diff
@@ -132,15 +137,20 @@
                # 113X: Databases
                1130 : 'EDB',
                1131 : 'EMYSQL',
-
+               # WMS/Workflow
+               1501 : 'EWMSJDL',
                # DMS/StorageManagement
                1601 : 'EFILESIZE',
                1602 : 'EGFAL',
                1603 : 'EBADCKS',
                1604 : "EFCERR",
 
                # RMS
-               1700 : ERMSUKN,
+               1700 : "ERMSUKN",
+
+               # TS
+               1900 : "ETSUNK",
+               1901 : "ETSDATA"
```

Comment:
One could add a comma at the end of this line, so next time somebody adds something they don't have to do it and confuse "git blame"